cms/static/sass/elements/_header.scss

```diff
@@ -74,6 +74,7 @@
   .nav-dd {
 
     .title {
+      @extend %t-demi-strong;
```

Comment:
How does this relate to focus and the purpose of the original ticket? This seems like its out of scope as well and should be made into its own commit/PR.